lib/presentation/redux/reducers/shared_space_destination_picker_reducer.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_destination_picker_state.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/shared_space_document_state.dart';
+import 'package:redux/redux.dart';
+
+final sharedSpaceDestinationPickerReducer = combineReducers<SharedSpaceDestinationPickerState>([
```

Comment:
IMO, it is `sharedSpaceDocumentDestinationPickerReducer`

Reply:
oke boy